apps/web/src/pages/EventPage/EventPage.tsx

```diff
@@ -107,98 +107,172 @@ export default function EventPage({
     }
   };
 
-  // The following code is here to keep a record of how to use the api calls
-
-  // const handleDelete = async () => {
-  //   if (eventData !== undefined && userRecord) {
-  //     await api.post(
-  //       '/events/delete',
-  //       JSON.stringify({ eventId: eventData.eventId })
-  //     );
-
-  //     console.log('Event deleted');
-  //     history.push('/calendar');
-  //   }
-  // };
-
-  // const handleRemove = async () => {
-  //   if (eventData !== undefined && userRecord) {
-  //     await api.post('/events/guests/delete', { eventId: eventData.eventId });
-
-  //     console.log('Event Guest removed');
-  //     history.push('/calendar');
-  //   }
-  // };
-
-  // const handleUpdateGuests = async () => {
-  //   if (eventData !== undefined && userRecord) {
-  //     await api.post('events/update-guests', {
-  //       eventId: eventData.eventId,
-  //       guestsByUserId: eventGuests
-  //         .filter(
-  //           (guest) =>
-  //             guest.status === EVENT_GUEST_STATUS.ACCEPTED ||
-  //             guest.status === EVENT_GUEST_STATUS.PENDING
-  //         )
-  //         .map((guest) => guest.uid),
-  //     });
-
-  //     console.log('Guests Updated');
-  //   }
-  // };
+  // api calls
+
+  const handleDelete = async () => {
+    if (eventData !== undefined && userRecord) {
+      await api.post(
+        '/events/delete',
+        JSON.stringify({ eventId: eventData.eventId })
+      );
+
+      console.log('Event deleted');
+      history.push('/calendar');
+    }
+  };
+
+  const handleRemove = async () => {
```

Comment:
user gets redirected to the 404 page when they try to remove themselves from the event, but it does remove them!

Reply:
Perhaps because we should be using `history.push('dashboard')` instead of `history.push('calendar')`

This should apply to both the handleDelete and handleRemove functions

Reply:
thank you! fixed now.